appengine/endpoints-frameworks-v2/discovery/src/main/java/com/example/helloendpoints/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.example.helloendpoints;
+
+/**
+ * Contains the client IDs and scopes for allowed clients consuming the helloworld API.
+ */
+public class Constants {
+  public static final String WEB_CLIENT_ID = "replace this with your web client ID";
```

Comment:
No I did not. I updated the README with information about Google Accounts Client Ids.